UNRELEASED.yaml

```diff
@@ -28,3 +28,9 @@ FIXED:
 #   FIXED:
 #     - bpk-component-horcrux:
 #       - Fixed issue where `BpkHorcrux` would occasionally possess the living.
+
+FIXED:
```

Comment:
Shouldn't this be a breaking change?

Reply:
It follows the same change we made for RN which was marked as not breaking

Reply:
Visually it will be breaking if someone has decided not to use the standard squared button